src/main/java/org/myrobotlab/swing/InMoovGui.java

```diff
@@ -64,6 +64,8 @@
   private final JPanel panel = new JPanel();
   private final JCheckBox Flip = new JCheckBox("Flip Camera");
   private final JCheckBox PyramidDown = new JCheckBox("PyramidDown");
+  private final JLabel lblNewLabel_1 = new JLabel("InMoov SwingGui is not yet ready... ");

```

Comment:
please use good variable names.  NewLabel_1 is not a good variable name.

Reply:
agree..